readme.md

```diff
@@ -1,7 +1,56 @@
-# MODiX - A moderation and analysis bot for Discord
+# MODiX - .NET C# Discord Bot
 
-Issues are the center of MODiX´s development. You can see what features are being worked on in the issues and you can see which features are planned. You might also create issues to share problems/ideas. 
+![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/discord-csharp/MODiX/build-container.yml)
+![Discord](https://img.shields.io/discord/143867839282020352)
+![GitHub License](https://img.shields.io/github/license/discord-csharp/MODiX)
 
-## Getting Started
+MODiX is a .NET C# Discord Bot that focuses on versatility, moderation and fun tools, primarily for users of the the largest [C# Discord](https://discord.gg/csharp) guild. MODiX's featureset is driven primarily by the C# guilds' needs, but feature requests, suggestions and other contributions are welcome and the bot is fully self hostable, and multi-tenant capable.
 
-Check out the wiki here: https://github.com/discord-csharp/MODiX/wiki/Developers
\ No newline at end of file
+MODiX's primary features:
+- C# REPL, to execute C# directly in Discord
+- Persistent user notes, warnings, muting and banning
+- Message quoting embeds
+- Custom inline message tags, acting as macros to send templated messages that frequently get mentioned
+
+If you want to report issues, discuss development or simply meet those who maintain MODiX, jump in on the dedicated [MODiX-development channel on Discord](https://discord.com/channels/143867839282020352/536023005164470303).
+
+## How to contribute
+
+### Development environment
+
+You will need the .NET 8 SDK installed. MODiX is developed in VS Code, Visual Studio and Rider to maximise developer satisfaction.
```

Comment:
I think it might be better to manually update the docs, especially if we move to preview versions, like we historically have done, where latest != preview for example?